test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -217,22 +222,21 @@
                                                        "foo-2" "bar"}))
 
       ;; Add two facts with the same :path, but different values.
-      (let [[p1 p2] (map :id (sql/insert-records
-                              :fact_paths
-                              {:value_type_id 0
-                               :path "orphan" :name "orphan" :depth 0}
-                              {:value_type_id 1
-                               :path "orphan" :name "orphan" :depth 0}))]
-        (sql/insert-records
-         :fact_values
-         {:path_id p1
-          :value_type_id 0
-          :value_hash (hash/generic-identity-hash "1")
-          :value_string "1"}
-         {:path_id p2
-          :value_type_id 1
-          :value_hash (hash/generic-identity-hash 1)
-          :value_integer 1}))
+      (let [[p1 p2]
+            (map :id (sql/insert! (jdbc/db) :fact_paths
+                                  {:value_type_id 0
+                                   :path "orphan" :name "orphan" :depth 0}
+                                  {:value_type_id 1
+                                   :path "orphan" :name "orphan" :depth 0}))]
+        (sql/insert! (jdbc/db) :fact_values
```

Comment:
can we just make our own `jdbc/insert!` which does this? Similar for `db-do-commands`? I think that's what @wkalt had in mind for the dynamic var.